examples/api/lib/widgets/gesture_detector/gesture_detector.2.dart

```diff
@@ -0,0 +1,109 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+void main() {
+  // Uncomment the following line to print information about gesture arenas.
+  // debugPrintGestureArenaDiagnostics = true;
+
+  runApp(const NestedGestureDetectorsApp());
+}
+
+class NestedGestureDetectorsApp extends StatelessWidget {
+  const NestedGestureDetectorsApp({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      home: Scaffold(
+        appBar: AppBar(title: const Text('Nested GestureDetectors')),
+        body: const NestedGestureDetectorsExample(),
+      ),
+    );
+  }
+}
+
+class NestedGestureDetectorsExample extends StatefulWidget {
+  const NestedGestureDetectorsExample({Key? key}) : super(key: key);
+
+  @override
+  State<NestedGestureDetectorsExample> createState() => _NestedGestureDetectorsExampleState();
+}
+
+class _NestedGestureDetectorsExampleState extends State<NestedGestureDetectorsExample> {
+  bool _isYellowTranslucent = false;
+  bool _greenTapped = false;
+  bool _yellowTapped = false;
+  final Border highlightBorder = Border.all(color: Colors.red, width: 5);
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: <Widget>[
+        Expanded(
+          child: GestureDetector(
+            onTap: () {
+              debugPrint('Green onTap');
+              setState(() {
+                // Both can't be tapped as there is only one winner in gesture arena
+                _yellowTapped = false;
+                _greenTapped = true;
```

Comment:
Should this (and the other handler) actually only set one variable? Right now, this doesn't actually illustrate that only one handler gets called. They could be called both as one handler would just override the behavior of the previous one.

If we do this, maybe also include a reset button in the sample to remove the boarders.

Reply:
In my last commit, I replaced this with an enum and used it to set only one variable. I also added a reset button.